fl4health/clients/mr_mtl_client.py

```diff
@@ -0,0 +1,188 @@
+from logging import INFO
+from pathlib import Path
+from typing import Dict, List, Optional, Sequence, Tuple
+
+import torch
+import torch.nn as nn
+from flwr.common.logger import log
+from flwr.common.typing import Config, NDArrays, Scalar
+
+from fl4health.checkpointing.checkpointer import TorchCheckpointer
+from fl4health.clients.basic_client import BasicClient
+from fl4health.parameter_exchange.full_exchanger import FullParameterExchanger
+from fl4health.utils.losses import LossMeterType, TrainingLosses
+from fl4health.utils.metrics import Metric
+
+
+class MrMtlClient(BasicClient):
```

Comment:
This is the correct capitalization scheme I think, but it keeps making me think we are writing "Mr. Montreal" in shorthand 😂

Reply:
Lol 😂